wordpress-mobile/WordPress-iOS
# Implement workaround for slow likes fetches in Stats

Fixes #11657, kinda-ish.

Alright, so this "fixes" the slow Stats summary fetches. 

As I explained in the WPKit PR, I tried almost everything I could think of to "fix" this, but this workaround was closest I could get to something reasonable. We could think about a nicer way of handling this in UI in a future update, but I think this should do for now...

To test:

1. Go to stats
2. Verify that the summary data appears much faster — with "likes" being set to 0
3. Verify that the correct likes number appears _eventually_ — or that there's a timeout error being printed in the console.
